tools.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
This is new in 1.17, I want to delete it before we fully switch to v1.17

Reply:
Removed.

Reply:
You haven't pushed the latest code yet?

Reply:
ping @tokers

Reply:
@tokers  any update?

Reply:
I have updated it but I don't have time to validate the e2e case until this weekend.